### ISSUE TITLE
Application is inaccessible to screen readers.

### ISSUE THREAD

**Issue by trypsynth** [JacoboGuijar/pdf-scraper-with-ocr#2]
Hello,
    This application uses Python's TKinter module, which renders it's controls in a way that makes it inaccessible to screen readers. There's a library called wxPython that doesn't have this issue; it renders everything natively.
    Since the GUI for this app isn't too advanced, switching to wxPython shouldn't be too hard. Can this be looked into please? This application would serve a great need among blind people (we dread anything having to deal with PDFs because they normally contain images).

**Comment by JacoboGuijar** [comment]
Hi, I wasn't aware of this issue until now, sorry for the inconveniences it may have cause you. I can assure you that updating the GUI is among my to do list with this project. I'll keep your comments in mind whenever I update it in the future. 

Currently I am quite busy with exams so I'll start to rebuild this in january or february. 

I'll keep you updated whenever the next version is ready.

Also, if you have any other comments about how this tool could be improved to facilitate the your use of the tool it would be highly appreciated. 

Regards.

**Comment by trypsynth** [comment]
Awesomeness! And no worries, a lot of people don't know about the brokenness of TK and screen readers. It draws all the controls and doesn't talk to accessibility APIs.
